app/views/pages/hiring_fundraiser.html.haml

```diff
@@ -0,0 +1,138 @@
+#page-hiring-job.bg-backgroundColorA.pb-16
+  %header.c-shapes
+    .md-container
+      = link_to hiring_page_path, class: "c-prominent-link" do
+        = graphical_icon "arrow-left"
+        Back to All Job Listings
+
+      .role
+        = graphical_icon "hiring-fundraiser", category: 'graphics', css_class: 'graphic'
+        .information
+          %h1.text-h0.mb-12 Head of Partnerships and Fundraising
+          .details
+            .remote-tag
+              = graphical_icon "stars"
+              100% remote
+            .salary Salary: £50k (~$68k, ~€60k)
+
+  %article
+    .sm-container
+      %p.text-p-2xlarge.mb-40
+        %strong Become Exercism's Head of Partnerships and Fundraising!
+        Are you a strategic thinker? Do you have a track record of generating revenue?
+        Do you care about high-quality, lifelong education, and the opportunities it provides?
+        Do you want to work with an exceptional team and help underpin a platform that touches millions of lives?
+        If so, this role might be perfect for you.
+
+      %h2.text-h2.mb-12 Summary
+      %p.text-p-xlarge.mb-16
+        Exercism is a not-for-profit organisation providing world-class programming education to anyone for free.
+        Our online platform is centred around self-directed learning and volunteer mentoring, and has nearly one million members.
+        Over the next 5 years, we want to become the defacto programming education platform that anyone, anywhere can use for free.
+        Through our work we aim to improve the quality of programming skills globally, and increase participation in tech from under-represented groups.
```

Comment:
Is "quality of skills" a thing? Isn't it just "improving skills"?

Reply:
Yeah. This one could be improved. I'm trying to say something along the lines of "By everyone getting better coding skills, the overall quality of programming increases". So "improve programming skills" feels a bit too much like we're improving individual people's skills, whereas I'm trying to say "improve the global standard of programming". Maybe that's a better phrasing?

Reply:
> improve the global standard of programming

Sounds good. Makes it a lofty goal.